board/miyoo/main/gmenu2x/sections/applications/usb-hid

```diff
@@ -0,0 +1,3 @@
+title=USB HID
+description=USB HID PC gamepad
+exec=/mnt/apps/usb-hid/usb-hid.sh
```

Comment:
We kinda missing left space in apps section, could you maybe rework TVON/TVOFF to one script e.g. TVout?